src/style.css

```diff
@@ -1 +1,68 @@
 /* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 1px 2px 4px rgba(0, 0, 0, 0.05);
+  cursor: pointer;
+  padding: 0 50px;
+  font-weight: 500;
+
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  list-style: none;
+
+}
+
+.logo {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+
+}
+
+.nav__link {
+  display: inline-block;
+  text-decoration: none;
+  padding: 23px 0;
+  color: #000;
+  white-space: nowrap;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
Move all repeatable colors to CSS variables.

[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)